src/htmx.js

```diff
@@ -1081,6 +1085,24 @@ return (function () {
             getInternalData(elt).sseEventSource = source;
         }
 
+        function processSSEForward(elt, sseEventName) {
+            var sseSourceElt = getClosestMatch(elt, hasEventSource);
+            if (sseSourceElt) {
+                var sseEventSource = getInternalData(sseSourceElt).sseEventSource;
+                var sseListener = function (event) {
+                    if (maybeCloseSSESource(sseSourceElt)) {
+                        sseEventSource.removeEventListener(sseEventName, sseListener);
+                        return;
+                    }
+                    triggerEvent(elt, sseEventName, event)
```

Comment:
Could we reuse the same code in processSSESwap for triggering the event?

```js
                    triggerEvent(elt, "htmx:sseMessage", event)
```

You would need to look into the `event.detail` object to see the event name, which I guess is a little annoying.

Alternatively, we could adopt this approach (using the SSE event name for the DOM event) in place of the current `htmx:sseMessage` approach in `processSSESwap()`

I don't have strong opinions on this.

Reply:
I think using htmx:sseMessage minimises the chances of a name collision.

Reply:
I've made the change.